lib/map.rb

```diff
@@ -1,36 +1,42 @@
 require 'geography'
+require 'chunky_png'
 
 class Map
   include Geography
 
   attr_accessor :grid
 
-  def initialize(dimensions)
-    @grid = generate_grid(dimensions)
+  def initialize
+    @grid = generate_grid
+  end
+
+  def height_map
+    @height_map ||= open_map
+  end
 
-    # TODO: get dimensions, elevation from height map file 
-    @height_map = nil
+  def open_map
+    # TODO: mechanism to choose height_map
+    ChunkyPNG::Image.from_file(Dir["lib/data/maps/*"].first)
```

Comment:
I've got some yaml loader code in calendar.rb, feel free to yoink that if it helps.